configs/milvus.yaml

```diff
@@ -286,6 +286,8 @@ proxy:
   ginLogging: true
   ginLogSkipPaths: / # skip url path for gin log
   maxTaskNum: 1024 # The maximum number of tasks in the task queue of the proxy.
+  ddlConcurrency: 16 # The concurrent execution number of DDL at proxy.
```

Comment:
do we really need a concurrency here?
can we send all request directly to rootcoord and skip the queue

Reply:
Make a concurrency limitation is make sense here.
All the read-only request like `describe/show/list` is set as ddl type now, and rate limit don't work on it.
We don't know the real load of rootcoord if just make no control at proxy.
So for safety, make a concurrency control at proxy-side.

The key problem is that we should define that what is ddl, and why some ddl opeartion is so slow.
I found that load collection opeartion should perform more asychoronously.
Just trigger a async task and return, wait for `getloadingstate` query to check the final state.